lib/maker_repository.rb

```diff
@@ -0,0 +1,26 @@
+class MakerRepository
+  def all
+    makers = []
+
+    # Send the SQL query and get the result set.
+    sql = 'SELECT id, name, genre FROM makers;'
```

Comment:
The SQL here references the column titles from the previous project, I believe the information that you want is id, name, username, email password